tasks/eva_2850/fix_discordant_variants.py

```diff
@@ -182,13 +196,13 @@ def create_merge_event(variant_merged, variant_retained):
     return merge_event
 
 
-def update_ss_with_new_rs(old_rs, new_rs):
+def update_ss_with_new_rs(old_rs, new_rs, assembly):
     dbsnp_sve_collection = mongo_source.mongo_handle[mongo_source.db_name][DBSNP_SUBMITTED_VARIANT_ENTITY]
     eva_sve_collection = mongo_source.mongo_handle[mongo_source.db_name][EVA_SUBMITTED_VARIANT_ENTITY]
 
     logger.info(f"updating submittedVariantEntity with old_rs: {old_rs} to new_rs: {new_rs}")
 
-    filter_query = {'rs': old_rs}
+    filter_query = {'rs': old_rs, 'seq': assembly}
```

Comment:
Is there a diagnostic/remediation needed to fix any SS that were incorrectly modified before this filter was added?  Or is this covered by the other script?